app/services/puppet_fact_parser.rb

```diff
@@ -95,10 +95,6 @@ def ipmi_interface
     Hash[ipmi].with_indifferent_access
   end
 
-  # Remove this method override when dropping support for facter < 3.0
```

Comment:
I think this comment is still valid. The whole function isn't used if we detect facter 3.0.

Reply:
Can you keep this?